{"name": "test_get_lab_results_not_found", "status": "passed", "description": "Test handling of no lab results found.", "attachments": [{"name": "log", "source": "a9e48f8f-8a9a-4fac-a9f9-c24ac2d7a54a-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "e6926dec-e25f-4850-965b-0fe210cdf7d4-attachment.txt", "type": "text/plain"}], "start": 1748170335365, "stop": 1748170335395, "uuid": "90f5c449-def8-4150-b363-de8a30d778bb", "historyId": "92c2d9005bb3914b49d4c41862a2e044", "testCaseId": "92c2d9005bb3914b49d4c41862a2e044", "fullName": "tests.api.test_api.TestLabResultsAPI#test_get_lab_results_not_found", "labels": [{"name": "feature", "value": "Lab Results API"}, {"name": "story", "value": "No Lab Results Found"}, {"name": "severity", "value": "normal"}, {"name": "parentSuite", "value": "tests.api"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "TestLabResultsAPI"}, {"name": "host", "value": "BardaTop"}, {"name": "thread", "value": "17888-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.test_api"}]}